<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMToolsClass</class>
 <widget class="QMainWindow" name="GMToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>716</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>GMTools</string>
  </property>
  <property name="windowOpacity">
   <double>8.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Courier New&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>620</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string>链接设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>470</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>IP地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ip"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>端口号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_port">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>用户UID：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_uid"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="top_btn_connect">
     <property name="geometry">
      <rect>
       <x>484</x>
       <y>26</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>链接服务器</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>620</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string>命令模式</string>
    </property>
    <widget class="QPushButton" name="center_btn_commit">
     <property name="geometry">
      <rect>
       <x>484</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="center_lineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>43</y>
       <width>464</width>
       <height>37</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>22</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>命令：</string>
     </property>
    </widget>
    <zorder>center_lineEdit</zorder>
    <zorder>label_4</zorder>
    <zorder>center_btn_commit</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>211</y>
      <width>301</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>快捷模式</string>
    </property>
    <widget class="QScrollArea" name="bottom_scroll_area">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="botton_scorill_area_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>262</width>
        <height>349</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="bottom_btn_del">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>380</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bottom_btn_add">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>380</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>211</y>
      <width>311</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>信息提示</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QListWidget" name="bottom_listWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>21</y>
       <width>301</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="separator"/>
    <addaction name="btn_help"/>
    <addaction name="separator"/>
    <addaction name="btn_about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="btn_help">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于GMTool</string>
   </property>
  </action>
  <action name="btn_about">
   <property name="text">
    <string>关于GMTool</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gmtools.qrc"/>
 </resources>
 <connections/>
</ui>
